script.harmony.control/addon.xml

```diff
@@ -0,0 +1,25 @@
+<addon id="script.harmony.control" name="Harmony Hub Controller" provider-name="pkscout" version="0.1.105">
```

Comment:
Why not use 0.1.5+matrix.1 and keep it in line with https://github.com/xbmc/repo-scripts/pull/1630. That is the preferred way.

Reply:
If it's OK, I can change the numbering for the next release. It's possible someone grabbed this from my beta repo, and if I change now they won't get updated (unless I bump the minor too, but then I also need to update the PR for the Leia version so they're in sync).

Reply:
In that case I would suggest you to use `0.1.105` for Leia and `0.1.105+matrix.1` for Matrix in these two PR's?